test/sql/detailed_profiler/test_detailed_profiler.test

```diff
@@ -0,0 +1,37 @@
+# name: test/sql/detailed_profiler/test_detailed_profiler.test
+# description: Detailed Profiler
+# group: [detailed_profiler]
+
+
+statement ok
+PRAGMA enable_profiling
+
+statement ok
+PRAGMA profiling_output='test.json'
```

Comment:
Please use `__TEST_DIR__/test.json` so we can keep test-generated files in the testing directory for easy clean up.